src/main/perl/Filesystem.pm

```diff
@@ -157,6 +157,66 @@ sub remove_if_needed
 
 =pod
 
+=head2 check_in_fstab
+
+Parse the fstab entry for the C<$self> filesystem. When a (PART)UUID is found, 
+this is used instead of the device name or label. 
+If it concerns a protected (i.e. never-modify) mountpoint or filesystem type,
```

Comment:
can you use the same terminology for protected/never-modify as in the fstab schema (i.e. is this keep or static or both?)